public/stylesheets/projects-style.css

```diff
@@ -4,6 +4,15 @@ body {
   overflow: hidden;
 }
 
+.project-list-wrapper {
+  position: fixed;
+  overflow: scroll;
```

Comment:
You don't see scrollbars? I do. They only show up if you have a mouse plugged in though, like other pages.